src/components/CommentsList.tsx

```diff
@@ -0,0 +1,21 @@
+import Comment from './Comment';
+import { classes } from '../style/comments.st.css';
+interface IPostOwner {
+    name: string;
+    profilePicture: string;
```

Comment:
Why does the commentsList receive a single profile picture?

pass this information in an object that represents a comments data (content, userImg, time, etc..) and render the Comment components in a repeater

Reply:
?? would like to understand what you meant

Reply:
MOVE ARRAY AS STATE TO PARENT